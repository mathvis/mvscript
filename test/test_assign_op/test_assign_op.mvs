let a = "test";
a = ("test" + a);
