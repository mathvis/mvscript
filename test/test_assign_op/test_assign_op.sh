#!/bin/bash
helix test_assign_op.mvs test_assign_op.out --vsplit
