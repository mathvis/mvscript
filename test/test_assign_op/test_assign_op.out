Decl (Variable (VarIdentifier "a") (Just string) (Just (Type (String "test"))))
Decl (Assignment (Assign (VarIdentifier "a") (Parentheses (Operation (Add (Type (String "test")) (VarIdentifier "a"))))))
